centinel/unit_test/http_unit_test.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+from  centinel.primitives import http
+import os
+
+
+class TestHTTPMethods:
+
+    def test_URL_NotExist(self):
```

Comment:
This does not conform to the method naming convention used in our code. Please revise.